libs/ledger/include/ledger/consensus/stake_manager_interface.hpp

```diff
@@ -29,18 +29,18 @@ class Address;
 }  // namespace chain
 namespace ledger {
 
-class Block;
-
 class StakeManagerInterface
 {
 public:
+  using BlockIndex = uint64_t;
```

Comment:
Minor: we might want to use BlockIndex = Block::Index rather than it be a given

Reply:
I did think about that but I wanted to try and keep coupling down